.gitignore

```diff
@@ -11,11 +11,5 @@ cache/
 coverage/
 .coverage_cache/
 coverage.json
-.yarn/*
-!.yarn/cache
-!.yarn/releases
-!.yarn/plugins
-!.yarn/sdks
-!.yarn/versions
-
+.yarn/
```

Comment:
as I already said this is not standard so until we don't decide to move to nx, let's keep it standard and commit what needs to be versioned (maybe we can just avoid committing cache if we don't want that, for more on this: https://github.com/yarnpkg/berry/issues/454)